c2corg_ui/static/js/auth/auth.js

```diff
@@ -151,10 +161,19 @@ app.AuthController.prototype.successRegister_ = function(response) {
  * @private
  */
 app.AuthController.prototype.errorRegister_ = function(response) {
-  // TODO
-  alert('register error');
-  console.log('register error');
-  console.log(response);
+  // TODO: i18n
+  // FIXME: HTML is not interpreted in alert messages
+  var msg = 'Registration failed because of:';
+  var errors = response['data']['errors'],
+      len = errors.length;
+  if (len > 0) {
+    msg += '<ul>';
```

Comment:
Maybe but that's not easy anyway since the alert system does not know what HTML code will be passed. In some cases we might want to display a link etc.